core/menuitem.js

```diff
@@ -113,8 +113,8 @@ MenuItem.prototype.createDom = function() {
   element.className = 'blocklyMenuItem goog-menuitem ' +
       (this.enabled_ ? '' : 'blocklyMenuItemDisabled goog-menuitem-disabled ') +
       (this.checked_ ? 'blocklyMenuItemSelected goog-option-selected ' : '') +
-      (this.highlight_ ?
-        'blocklyMenuItemHighlight goog-menuitem-highlight ' : '') +
+      (this.highlight_ ? 'blocklyMenuItemHighlight goog-menuitem-highlight ' :
+                         '') +
```

Comment:
It's weird that clang does horizontal alignment, when that is [explicitly discouraged](https://engdoc.corp.google.com/eng/doc/devguide/js/styleguide.md?cl=head#formatting-horizontal-alignment). Probably fine to leave it as is, just wanted to point it out =)

Reply:
Good observation.  This is certainly horizontal alignment by the definition given in the style guide.  I note, however, that it is also quite different from the example given there, which is more tabular in nature and is statement-to-statement.  Here it is intra-statement (so less subject to the "blast radius" concern, since any change which would mangle the alignment might also force a change to the wrapping anyway), and—IMHO—helps make the ternary more readable.